src/AmalgamatorSession.ts

```diff
@@ -382,10 +383,14 @@ export class AmalgamatorSession extends LoggingDebugSession {
         const childResponse = await childDap.stackTraceRequest(args);
         const frames = childResponse.body.stackFrames;
         // XXX: When does frameHandles get reset as we don't have a "stopped all"
-        frames.forEach(
-            (frame) =>
-                (frame.id = this.frameHandles.create([childDap, frame.id]))
-        );
+        frames.forEach((frame) => {
+            frame.id = this.frameHandles.create([childDap, frame.id]);
+            if (frame.instructionPointerReference) {
+                frame.instructionPointerReference = this.disassembleHandles
```

Comment:
Rather than use handles, prefix the `frame.instructionPointerReference` with the `${this.index}:`.

Reply:
This comment seems not to have been addressed as you are still using handles.

Reply:
I'm going to mark this as resolved as there are now other comments that say to remove handles.